src/module.c

```diff
@@ -5972,6 +5972,18 @@ robj **moduleCreateArgvFromUserFormat(const char *cmdname, const char *fmt, int
  *              after the command invocation (without releasing the Redis lock in between).
  *              The module should not keep the promise call reply after the Redis lock has been released.
  *              The module should not free the promise call reply.
+ *              Notice that on unblocking, the only promise is that the unblock handler will be called,
+ *              If the blocking RM_Call caused the module to also block some real client (using RM_BlockClient),
+ *              it is the module responsibility to unblock this client on the unblock handler.
+ *              On the unblock handler it is only allow to perform the following:
+ *              * Calling more Redis commands using RM_Call
+ *              * Open keys using RM_OpenKey
+ *              * Replicate data to the replica or AOF
+ *              Specifically, it is not allowed to call any Redis module API which are client related such as:
+ *              * RM_Reply* API's
+ *              * RM_BlockClient
+ *              * RM_GetCurrentUserName
```

Comment:
shouldn't these be indented more?

Reply:
ohh, my bad. for some reason i thought it's a sub-section of the bullet above.
i now see it's a separate list, so i think we need a blank line in between the end of the previous list and the header of the next one.

Reply:
Done